.editorconfig

```diff
@@ -0,0 +1,124 @@
+# To learn more about .editorconfig see https://aka.ms/editorconfigdocs
+############################### 
+# Core EditorConfig Options   # 
+############################### 
+root = true 
+# All files 
+[*] 
+indent_style = space 
+# Code files 
+[*.{cs,csx,vb,vbx}] 
+indent_size = 4 
+insert_final_newline = true 
+charset = utf-8-bom 

```

Comment:
utf-8 NO bom

Reply:
oh bom is good, there is one example of it, some characters usually are lost like:
- About.cs: copyright character
- RoundCorners.cs: quotation marks

Reply:
BOM has historically been bad for us. We had to remove it from files because they kept getting corrupted.
BOM is mostly a Windows thing Linux does not use them.